### DISCUSSION TITLE
ABout ImageMagick source code.

### DISCUSSION THREAD

**Discussion by pprasanthk** [ImageMagick/ImageMagick#7809] [Development]
### ImageMagick version

7.1.1-41

### Operating system, version and so on

Windows

### Description

Hi, 

Previously, we used to download ImageMagick source code from **https://imagemagick.org/archive/windows/releases/**. 
We used to unzip the downloaded zip file, we used to see the Configure folder. We used to build Configure.sln and run the generated Configure.exe to produce ImageMagick actual solution. Upon building this source code, we used to get the entire list of binaries. 

But, now the entire source code is available only for the main branch **ImageMagick-Windows-main.zip** but not for the individual releases, specifically Configure folder is not available in the branches of the individual releases source codes from **https://imagemagick.org/archive/windows/releases/**

Could you please let us know where to find the entire source code for the ImageMagick branches (for example, 7.1.1-41)?

--
Thanks
Prasanth

**Comment by dlemstra** [comment]
You can use this repository directly: https://github.com/ImageMagick/ImageMagick-Windows. This will always be the latest main. You could do a specific release of ImageMagick and the dependencies at that point in time. This is our own process so it might break in the future and that is why I will not explain how you can do a specific tag. You will need to figure that out yourself.
